.idea/misc_diff.java

```diff
@@ -0,0 +1,1374 @@
+//                 mEdit2=(EditText)findViewById(R.id.editTextNumber3);
+//                 String m1=mEdit2.getText().toString();
+//                 mEdit3=(EditText)findViewById(R.id.editTextNumber6);
+//                 String m2=mEdit3.getText().toString();
+//                 mEdit4   = (EditText)findViewById(R.id.editTextNumber4);
+//                 String y1=mEdit4.getText().toString();
+//                 mEdit5   = (EditText)findViewById(R.id.editTextNumber7);
+//                 String y2=mEdit5.getText().toString();
+
+//                 String day2="0";
+//                 System.out.println(m2+"yesssss     ");
+//                 Bundle b = new Bundle();
+//                 b.putString("m1",m1);
+//                 b.putString("y1",y1);
+//                 b.putString("d1",day2);
+//                 b.putString("m2",m2);
+//                 b.putString("y2",y2);
+//                 b.putString("d2",day2);
+
+//                 c.putExtras(b);
+//                 startActivity(c);
+//             }
+
+//         });
+//     }
+// }
+
+// Daywise
+
+//  package com.xa.xpensauditor;
+
+// import static java.lang.System.currentTimeMillis;
+
+// import androidx.annotation.NonNull;
+// import androidx.appcompat.app.AppCompatActivity;
+// import androidx.recyclerview.widget.RecyclerView;
+
+// import android.os.Bundle;
+// import android.util.Log;
+// import android.widget.Button;
+// import android.widget.DatePicker;
+// import android.widget.TextView;
+
+// import com.anychart.AnyChart;
+// import com.anychart.AnyChartView;
+// import com.anychart.chart.common.dataentry.DataEntry;
+// import com.anychart.chart.common.dataentry.ValueDataEntry;
+// import com.anychart.charts.Cartesian;
+// import com.anychart.core.cartesian.series.Column;
+// import com.anychart.enums.Anchor;
+// import com.anychart.enums.HoverMode;
+// import com.anychart.enums.Position;
+// import com.anychart.enums.TooltipPositionMode;
+// import com.firebase.client.Firebase;
+// import com.google.firebase.auth.FirebaseAuth;
+// import com.google.firebase.database.DataSnapshot;
+// import com.google.firebase.database.DatabaseError;
+// import com.google.firebase.database.DatabaseReference;
+// import com.google.firebase.database.FirebaseDatabase;
+// import com.google.firebase.database.ValueEventListener;
+
+// import java.util.ArrayList;
+// import java.util.Arrays;
+// import java.util.Calendar;
+// import java.util.Collections;
+// import java.util.Date;
+// import java.util.GregorianCalendar;
+// import java.util.HashMap;
+// import java.util.List;
+// import java.util.Map;
+// import java.util.TreeMap;
+
+// public class Day_wise_visualization extends AppCompatActivity {
+//     private Firebase mRootRef;
+//     private Firebase RefUid,RefTran,RefUid1;
+//     private Firebase RefName,RefEmail,RefPhnnum;
+//     private Firebase RefCat,RefFood,RefHealth,RefTravel,RefEdu,RefBills,RefHomeNeeds,RefOthers,RefUncat;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 107).